Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/InvoluntaryProject/SearchSchool.cshtml.cs

```diff
@@ -0,0 +1,55 @@
+using Dfe.PrepareConversions.Data.Services;
+using Dfe.PrepareConversions.Services;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Dfe.PrepareConversions.Pages.InvoluntaryProject
+{
+   public class SearchSchoolModel : PageModel
+	{
+		private readonly IGetEstablishment _getEstablishment;
+		private readonly ErrorService _errorService;
+
+		public SearchSchoolModel(IGetEstablishment getEstablishment, ErrorService errorService)
+		{
+			_getEstablishment = getEstablishment;
+			_errorService = errorService;
+		}
+
+		[BindProperty(Name = "query", SupportsGet = true)]
+		public string SearchQuery { get; set; } = "";
+
+		public IActionResult OnGet(bool change = false)
+		{
+			ViewData["ChangeLink"] = change;
```

Comment:
Where is this used?

Reply:
Its not, but suspect it will be later. Will take it out.